daskernetes/tests/conftest.py

```diff
@@ -8,8 +8,5 @@ def pytest_addoption(parser):
 
 @pytest.fixture
 def image_name(request):
-    worker_image = request.config.getoption('--worker-image')
-    if not worker_image:
-        pytest.fail("Need to pass --worker-image. Image must have same python setup as host!")
-        return
+    worker_image = request.config.getoption('--worker-image') or 'daskdev/dask:latest'
```

Comment:
Any objection with defaulting to `daskdev/dask`?

Reply:
In general, I think what we should do is:

1. Default those to None
2. Read from env variables as secondary default if they are None

I don't think we should use daskdev/dask as default, because:

1. It defaults to 'daskdev/dask:latest', and using latest anywhere is a ticking time bomb
2. It also will only work for people on the same version of python from conda.

I agree that users not having to know about images is a requirement for good UX. My suggestion is:

1. Default from something like DASKERNETES_CONTAINER_IMAGE
2. In JupyterHub deployment, set that to the same value as singleuser.image 

This would mean you have a single source of truth for image, and it won't drift out of use